README.md

```diff
@@ -4,54 +4,29 @@
 
 ## Table of contents
 
-- [How to setup your dev environment with Docker](#setup-your-dev-environment-with-docker),
-- [How to use Invoke tasks](#invoke-tasks),
+- [How to setup your dev environment with Docker](#setup-your-dev-environment-with-docker)
 - [Basket donations queue](#basket)
 
 ## Documentation
 
 - [Pages](docs/pages.md)
 - [Pontoon Integration](docs/pontoon_integration.md)
+- [Local dev](./docs/local_dev.md)
```

Comment:
should this be `/docs/local_dev.md` so it's consistent with the two links above?